test/replcompletions.jl

```diff
@@ -663,59 +666,57 @@ c, r, res = test_complete(s)
 @test isempty(c)
 
 # test Dicts
-test_dict = Dict("abc"=>1, "abcd"=>10, :bar=>2, :bar2=>9, Base=>3, contains=>4, `ls`=>5,
-      66=>7, 67=>8, ("q",3)=>11, "α"=>12, :α=>13)
-s="test_dict[\"ab"
+s="CompletionFoo.test_dict[\"ab"
```

Comment:
The same applies to almost all other tests in the same file though (this was basically the only one that uses a non-qualified name in the replcompletion test before). It's mainly to avoid polluting the namespace but making the name slightly longer could also be fine.